bundles/binding/org.openhab.binding.isy/src/main/java/org/openhab/binding/isy/internal/ErrorHandler.java

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.isy.internal;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.universaldevices.common.UDClientStatus;
+import com.universaldevices.resources.errormessages.ErrorEventListener;
+import com.universaldevices.resources.errormessages.Errors;
+import com.universaldevices.upnp.UDControlPoint;
+import com.universaldevices.upnp.UDProxyDevice;
+
+/**
+ * Error handler for the ISY client
+ * 
+ * @author Tim Diekmann
+ * 
+ */
+public class ErrorHandler implements ErrorEventListener {
+
+	private final Logger logger = LoggerFactory.getLogger(getClass());
+
+	private int socket_open_failed_count = 0;
+
+	private void offLine(final int status, Object device) {
+		String err = Errors.getErrorMessage(status);
+
+		this.logger.warn("Warning: " + status + (err == null ? " " : err));
+
+		if (UDClientStatus.isBusy()) {
+			return;
+		}
+		if (status == 1050) {
+			this.socket_open_failed_count++;
+		}
+		if (this.socket_open_failed_count < 3) {
+			return;
+		}
+		this.socket_open_failed_count = 0;
+
+		if (device == null) {
+			device = UDControlPoint.firstDevice;
+		}
+		if (device != null && device instanceof UDProxyDevice) {
+			UDControlPoint.getInstance().offLine((UDProxyDevice) device);
+		}
+
+	}
+
+	private void humanInterventionRequired(final int status, final String msg) {
+		String err = Errors.getErrorMessage(status);
+		this.logger.error("Human Intervention Required: " + status
+				+ (err == null ? " " : err));
+		if (msg != null) {
+			this.logger.error(msg);
+		}
+	}
+
+	private void warning(final int status, final String msg) {
+		String err = Errors.getErrorMessage(status);
+
+		this.logger.warn("Warning: " + status + (err == null ? " " : err));
+		if (msg != null) {
+			this.logger.warn(msg);
+		}
```

Comment:
It could be, wanted to clearly log that it was a issue and then a issue without inserting newlines in a error statement.